src/main/java/org/jenkinsci/plugins/githubautostatus/notifiers/InfluxDbNotifierSchemas.java

```diff
@@ -511,10 +511,10 @@ public String formatTestCase(String jobName,
                 return new InfluxDbLineBuilder(SeriesNames.TestCase)
                         .appendTagValue(TagNames.Owner, owner)
                         .appendTagValue(TagNames.Repo, repo)
+                        .appendTagValue(TagNames.Suite, suite)
```

Comment:
I wonder if we should rename this. I'm pretty sure that someone who upgrades will will have problems, since they'll already have a field named `suite`. See https://github.com/influxdata/influxdb/issues/4630

Reply:
Will rename it